test/spec/ol/renderer/canvas/replay.test.js

```diff
@@ -140,21 +141,33 @@ describe('ol.render.canvas.ReplayGroup', function() {
       expect(beginPathCount).to.be(3);
     });
 
-    it('applies the pixelRatio to the linedash array', function() {
-      var lineDash, lineDashCount = 0;
+    it('applies the pixelRatio to the linedash array and offset', function() {
+      var lineDash, lineDashCount = 0,
+          lineDashOffset, lineDashOffsetCount = 0;
 
       context.setLineDash = function(lineDash_) {
         lineDashCount++;
         lineDash = lineDash_.slice();
       };
 
+      Object.defineProperty(context, 'lineDashOffset', {
```

Comment:
`Object.defineProperty()` works on IE9, but with some limitations that are not relevant here.

Do you suggest another way to test that ?

By the way, there are several other tests that fail on IE9, while this one works.